tests/test_deployments/test_app.py

```diff
@@ -193,10 +193,6 @@ class TestTendermintServerApp(BaseTendermintServerTest):
     def test_files_exist(self) -> None:
         """Test that the necessary files are present"""
 
-        def remove_prefix(text: str, prefix: str) -> str:
-            """str.removeprefix only from python3.9 onward"""
-            return text[text.startswith(prefix) and len(prefix) :]
```

Comment:
not used?

Reply:
exactly 09f223c37743a45a7ee4ab4e7d765b0ae2d3b09e